starters/blog/package-lock.json

```diff
@@ -891,7 +891,7 @@
     },
     "@types/debug": {
       "version": "0.0.29",
-      "resolved": "https://registry.npmjs.org/@types/debug/-/debug-0.0.29.tgz",
+      "resolved": "http://registry.npmjs.org/@types/debug/-/debug-0.0.29.tgz",
```

Comment:
Hmm.. wonder why this change?

Reply:
Only updating gatsby dependency now and letting Renovate handle updates to starters